Dockerfile

```diff
@@ -1,18 +1,23 @@
-FROM ocrd/core:edge
+FROM ocrd/core
 MAINTAINER OCR-D
 ENV DEBIAN_FRONTEND noninteractive
 
+ENV PREFIX=/usr/local
+
 WORKDIR /build
 COPY ocrd-im6convert .
 COPY ocrd-tool.json .
 COPY Makefile .
 
 RUN apt-get update && \
+    apt-get -y install apt-utils && \
```

Comment:
Why is `apt-utils` needed?

Reply:
IIRC you need that for proper post-installation

Reply:
The error message you get without it is:
```
debconf: delaying package configuration, since apt-utils is not installed
```